test/Chainweb/Test/Rosetta/RestAPI.hs

```diff
@@ -209,49 +230,136 @@ blockTests tio envIo = testCaseSchSteps "Block Tests" $ \step -> do
     validateTransferResp bh resp1
   where
     req h = BlockReq nid $ PartialBlockId (Just h) Nothing
+    transferGasCost = gasCost 547
 
     validateTransferResp bh resp = do
       _blockResp_otherTransactions resp @?= Nothing
 
-      let validateBlock b = do
+      let validateBlock someBlock = do
+            Just b <- pure $ someBlock
             _block_metadata b @?= Nothing
             _blockId_index (_block_blockId b) @?= bh
             _blockId_index (_block_parentBlockId b) @?= (bh - 1)
 
             case _block_transactions b of
-              [x,y] -> case _transaction_operations x <> _transaction_operations y of
-                [a,r1, r2, b',c,d,e,f] -> validateTxs (Just (r1,r2)) a b' c d e f
-                [a,b',c,d,e,f] -> validateTxs Nothing a b' c d e f
-                _ -> assertFailure "total tx # should be >= 6: coinbase + possible remeds + 5 for every tx"
-              _ -> assertFailure "every block should result in at least 2 transactions: coinbase + txs"
+              [x,r1,r2,y] -> do
+                -- ^ coin v2 remediation block.
+                -- No coin table remediation for this version.
+                let ops = _transaction_operations x <> _transaction_operations r1 <>
+                          _transaction_operations r2 <> _transaction_operations y
+                case ops of
+                  [a,b',c,d,e,f] -> validateTxs Nothing a b' c d e f
+                  _ -> assertFailure "should have 6 ops: coinbase + 5 for transfer tx"
+
+              [x,r1,y] -> do
+                -- ^ 20 chain remediation block
+                let ops = _transaction_operations x <> _transaction_operations r1 <>
+                          _transaction_operations y
+                case ops of
+                  [a,rop1, b',c,d,e,f] -> validateTxs (Just rop1) a b' c d e f
+                  _ -> assertFailure "should have 7 ops: coinbase + 20 chain rem + 5 for transfer tx"
+
+              [x,y] -> do
+                -- ^ not a remediation block
```

Comment:
-- ^ won't pass CI I don't think. It's haddock-specific syntax in the wrong position.

Reply:
What do you mean?

Reply:
`-- ^` is special haddock syntax, which is reserved for top-level declarations. When we go to build docs, I believe that syntax will fail.

Reply:
aw, I've used `-- ^`  in other places as well (mainly in documenting specific type's fields). Will it break in those cases as well?

Reply:
possibly. if you build the docs you can see.

Reply:
Do you know how to build the docs? I tried `cabal haddock` and it just built the docs for the libraries